scripts/uninstall.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+PROGRAM_NAME="hzc"
+HZCLI_HOME="$HOME/.local/share/hz-cli"
+
+rm -f $HOME/.local/bin/$PROGRAM_NAME
+if [[ -r $HOME/.zshrc ]]; then
+    echo "You can remove ~/.local/bin from your path by removing a line in the form \"PATH=\$HOME/.local/bin:\$PATH\" from ~/.zshrc"
+    echo
+fi
+if [[ -r $HOME/.bashrc ]]; then
+    echo "You can remove ~/.local/bin from your path by removing a line in the form \"PATH=\$HOME/.local/bin:\$PATH\" from ~/.bashrc"
+    echo
+fi
+
+rm -rf ${HZCLI_HOME} #remove hzcli related files (config.yaml,.hzc.yaml,autocompletion scripts)
```

Comment:
This is OK for this prototype, but in the next one we should consider keeping the configuration in case the user modified it.